server/users.go

```diff
@@ -62,12 +69,13 @@ func (r *userRequest) ValidRoles() error {
 }
 
 type userResponse struct {
-	Links    selfLinks         `json:"links"`
-	ID       uint64            `json:"id,string"`
-	Name     string            `json:"name"`
-	Provider string            `json:"provider"`
-	Scheme   string            `json:"scheme"`
-	Roles    []chronograf.Role `json:"roles"`
+	Links      selfLinks         `json:"links"`
+	ID         uint64            `json:"id,string"`
+	Name       string            `json:"name"`
+	Provider   string            `json:"provider"`
+	Scheme     string            `json:"scheme"`
+	SuperAdmin bool              `json:"superAdmin"`
```

Comment:
Omitempty or dont' have the field?

Make sure that only superAdmins see this if it is true.

Reply:
Added to https://github.com/influxdata/chronograf/issues/2207